<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 85, 85);
font: 700 9pt &quot;Segoe UI&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="textbox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>371</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(67, 67, 67);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clr_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 30, 30);</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="sqrt_btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>âˆšx</string>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="onex_btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="fa_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>x!</string>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="log_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pm_btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="prcnt_btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="div_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="sin_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="cos_btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="tan_btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="cot_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>cot</string>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="mul_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
    <property name="icon">
     <iconset theme="accessories-calculator"/>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="min_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="plus_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="eq_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>340</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 186, 186);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 186, 186);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 186, 186);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 186, 186);</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 186, 186);</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 186, 186);</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 186, 186);</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 186, 186);</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 186, 186);</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dot_btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 186, 186);</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 186, 186);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
